samples/highcharts/chartchooser/continuous-comparison-map/demo.html

```diff
@@ -0,0 +1,5 @@
+<script src="https://code.highcharts.com/highcharts.js"></script>
```

Comment:
This demo doesn't run because it doesn't load Highcharts Maps

Reply:
Done 👍